docs/gradle-getting-started.md

```diff
@@ -224,40 +231,42 @@ In this step you'll learn how to propagate a local change to the application thr
 Modify `build.gradle` to include a dependency for `jscience` in the `dependencies` section like this:
 
 ```groovy
-:::>> file.append build.gradle#16
+:::>> file.append build.gradle#30
 compile "org.jscience:jscience:4.3.1"
 ```
 
 Modify `src/main/java/Main.java` so that it imports this library at the start, by including the following imports:
 
 ```java
-:::>> file.append src/main/java/Main.java#1
+:::>> file.append src/main/java/com/example/heroku/HerokuApplication.java#19
 import static javax.measure.unit.SI.KILOGRAM;
 import javax.measure.quantity.Mass;
 import org.jscience.physics.model.RelativisticModel;
 import org.jscience.physics.amount.Amount;
 ```
 
-And modify the `get("hello",...)` handler so that it reads like this:
+Add the following `hello ` method to `HerokuApplication.java`:
 
 ```java
-:::>> file.append src/main/java/Main.java#42
-.get("hello", ctx -> {
-  RelativisticModel.select();
-  Amount<Mass> m = Amount.valueOf("12 GeV").to(KILOGRAM);
-  ctx.render("E=mc^2: 12 GeV = " + m.toString());
-})
+:::>> file.append src/main/java/com/example/heroku/HerokuApplication.java#59
+@RequestMapping("/hello")
+String hello(Map<String, Object> model) {
+    RelativisticModel.select();
+    Amount<Mass> m = Amount.valueOf("12 GeV").to(KILOGRAM);
+    model.put("science", "E=mc^2: 12 GeV = " + m.toString());
+    return "hello";
+}
 ```  
 
 [Here's the final source code](https://github.com/heroku/gradle-getting-started/blob/localchanges/src/main/java/Main.java) for `Main.java` - yours should look similar.  [Here's a diff](https://github.com/heroku/gradle-getting-started/compare/localchanges) of all the local changes you should have made.
```

Comment:
This path also incorrect (as Main.java moved)

Reply:
Fixed in ca9a552